apis/r/R/TileDBGroup.R

```diff
@@ -60,22 +65,25 @@ TileDBGroup <- R6::R6Class(
         self$tiledb_timestamp
       }
       if (is.null(private$.group_open_timestamp)) {
-        spdl::debug("Opening {} '{}' in {} mode", self$class(), self$uri, mode)
-        private$.tiledb_group <- tiledb::tiledb_group(
-          self$uri,
-          type = mode,
-          ctx = private$.tiledb_ctx
-        )
+        spdl::debug("[TileDBGroup$open] Opening {} '{}' in {} mode", self$class(), self$uri, mode)
+        #private$.tiledb_group <- tiledb::tiledb_group(self$uri, type = mode, ctx = private$.tiledb_ctx)
+        #print(str(private$.soma_context))
+        private$.tiledb_group <- c_group_open(self$uri, type = mode, ctx = soma_context())#private$.some_context)
+        #group_open()
```

Comment:
Can we clean out the commented code?
```suggestion
        spdl::debug("[TileDBGroup$open] Opening {} '{}' in {} mode", self$class(), self$uri, mode)
        private$.tiledb_group <- c_group_open(self$uri, type = mode, ctx = soma_context())#private$.some_context)
```